### ISSUE TITLE
missing MSVCP120D.dll

### ISSUE THREAD

**Issue by ayellowpaper** [L0rdCha0s/FBXJointRenamer#3]
So I really want to use your program, cause it would save me sooo many hours and it looks great, but I'm having trouble. The MSVCP120D.dll is missing, which apparently is part of visual studio 2013 debug redistributables, which I don't have. And it's apparently very hard to come by in 2021. Is there a potential fix for this? I'd love any kind of direction

**Comment by L0rdCha0s** [comment]
Might be easier just to rebuild in visual studio community?  Otherwise I can probably update the build for you..

**Comment by ayellowpaper** [comment]
I tried to rebuild it, but that just lead to build errors also relating to the fact that it was build with VS 2013. I went down a rabid hole to find out how to fix it, but Microsoft makes it really hard.
You're an angle for offering to rebuild it. I just noticed though that I also need the functionality to remove nodes/bones from inbetween the skeleton, so I guess I'll have to get it running on my end

**Comment by NonReflectiveMirror** [comment]
Just a heads up.  I had the same issue so downloaded the two missing DLL files (32 bit versions MSVCP120D.dll and MSVCR120D.dll) from https://www.dll-files.com and copied them into the Debug directory and it's running fine.  They tested Virus free.  Haven't configured a jointmap yet but the executable runs and generates an output.fbx file.

A big thankyou to the dev if this works well, as I have a number of files to convert and this should be a big time saver.

**Comment by miklem** [comment]
I also want to run this, download dlls form link above, but on win 10 it`s not running even in compatible win7 mode, I`ve download VS2017, installed FBX SDK, but it can`t compile. I retarget project to new version

Upgrading project 'FBXTest'...
	Configuration 'Debug|Win32': changing Platform Toolset to 'v141' (was 'v120').
	Configuration 'Debug|x64': changing Platform Toolset to 'v141' (was 'v120').
	Configuration 'Release|Win32': changing Platform Toolset to 'v141' (was 'v120').
	Configuration 'Release|x64': changing Platform Toolset to 'v141' (was 'v120').
Retargeting End: 1 completed, 0 failed, 0 skipped

But once I start build I get a bunch of errors like
E1696	cannot open source file "fbxsdk.h"	
C1083	Cannot open include file: 'fbxsdk.h': No such file or directory	
and 78 others.

@L0rdCha0s could you recompile your tool for win 10, so we can just run it, since this tool is really needed.

**Comment by NonReflectiveMirror** [comment]
I would agree a freshly built executable and any dependant files would be the best solution if possible, however as I previously reported I am running it fine and that's on Win 10 without running in any compatability mode, so not sure why thats not running for you.  The only thing I did was avoid putting it in Program Files (x86) as it didn't have permissions to generate its output files there.

What happens when you attempt to run it with the two 32 bit DLLs present in the executable directory?  Do you get further errors reported? 

It's fully tested now with mappings in the cfg file and it works really well.

**Comment by miklem** [comment]
it just tell that application was unable to start correctly. That`s all.

**Comment by NonReflectiveMirror** [comment]
I had that when using the 64 bit DLLs which I downloaded first.  I seem to recollect there were multiple versions of each DLL too on that site, so here are the versions of the 32 bit one's I'm using that definitely work for me:

msvcp120d.dll version 12.0.20827.3
msvcr120d.dll  version 12.0.21005.1

both of which you can check against your files properties/details tab.

If yours don't match then try downloading those, otherwise you could try installing the Microsoft C++ Runtime Library 2013 which I believe these DLLs come from (but probably not the debug versions) incase there are other dependancies.  I already had that installed.
